lms/templates/instructor/instructor_dashboard_2/special_exams.html

```diff
@@ -10,22 +10,22 @@ <h6 class = "hd hd-6 escalation-email-container">${_('Proctortrack Escalation Em
   % endif
   <div id="proctoring-accordion">
     <div class="wrap">
-      <h3 class="hd hd-3">${_('Allowance Section')}</h3>
+      <h3 class="hd hd-3"><div class="special-exam-header">${_('Allowance Section')}</div></h3>
```

Comment:
can we add the `special-exam-header` class as an additional class for the h3 tag as opposed to creating a new div?